### PR TITLE
Fix/design fixes on blacklist screen

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[Design fixes on blacklist screen](https://trello.com/c/V2NcndKR/831-design-fixes-on-blacklist-screen)

File: new-lamassu-admin/src/pages/Blacklist/BlacklistModal.js [hunk 1]
```diff
@@ -74,9 +75,11 @@ const BlackListModal = ({
         </Form>
       </Formik>
       <div className={classes.footer}>
-        <Link type="submit" form="address-form">
-          Blacklist address
-        </Link>
+        <Box display="flex" justifyContent="flex-end">
+          <Link type="submit" form="address-form">
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Blacklist/BlacklistModal.js:79]: I'm not really a fan of inlining styles as props. This is an inconsistency we have thoughout some components in our code base. IMO, we should always keep styles contained in styling files

**Review by chaotixkilla — APPROVED** [review]

File: new-lamassu-admin/src/pages/Blacklist/Blacklist.js [hunk 2]
```diff
@@ -120,8 +121,20 @@ const Blacklist = () => {
...
   }
 
+  const validateAddress = (cryptoCode, address) => {
+    try {
+      return !R.isNil(coinUtils.parseUrl(cryptoCode, 'main', address))
+    } catch {
+      return false
+    }
+  }
+
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Blacklist/Blacklist.js:131]: I think you can add this as a test on the yup schema, there's no need for it to be outside the modal.

File: new-lamassu-admin/src/pages/Blacklist/BlacklistModal.js (hunk 1 above)
Comment (RafaelTaranto) [reply new-lamassu-admin/src/pages/Blacklist/BlacklistModal.js:79]: Ohh, this box thing is made to simplify flexbox behavior. This is not really inline styling, just props being passed to the component.

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**